Deep_Learning/Dog Breeds Classification/web-app/src/App.css

```
@@ -0,0 +1,129 @@
+} */
+
+.App {
+  margin: 5% auto;
+  width: calc(100vw - 50%);
+  height: calc(100vh - 20%);
+  text-align: center;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  /* background-image: url("./wallp.jpeg"); */
+  /* background-color: rgba(0, 0, 0, 0.4); */
+  background-size: cover;
+  background: #f4ff91;
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-size



(lint/suspicious/noShorthandPropertyOverrides)